pkg/cnb/create_builder.go

```diff
@@ -90,9 +90,11 @@ func (r *RemoteBuilderCreator) CreateBuilder(keychain authn.Keychain, clusterSto
 		Buildpacks:              buildpackMetadata(builderBldr.buildpacks()),
 		Order:                   builderBldr.order,
 		ObservedStackGeneration: clusterStack.Status.ObservedGeneration,
-		ObservedStoreGeneration: clusterStore.Status.ObservedGeneration,
+		ObservedStoreGeneration: fetcher.ClusterStoreObservedGeneration(),
 		OS:                      config.OS,
-	}, nil
+	}
+
+	return fetcher.UsedObjects(), builder, nil
```

Comment:
Do we need to return the `fetcher.UsedObjects()` if they are not used in the builder or clusterbuilder reconcilers anymore?

Reply:
Thought about it some more, and the only information we would need for exit-early is already present in the builder spec, so we can get rid of `UsedObjects()` entirely